### PR TITLE
[front] - chore(agents): throw error when running assistant w/ disabled provider

### LINKED ISSUE #5544: [Model Deactivation] - Throw error when using agent with unsupported model
We want to throw an error when running an assistant with a disabled provider.

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at throwing an error when running assistants with a disabled model provider.

## Risk

Breaking the conversation flow

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/lib/api/assistant/conversation.ts [hunk 1]
```diff
@@ -645,6 +649,22 @@ export async function* postUserMessage(
...
+  const whiteListedProviders = owner.whiteListedProviders ?? MODEL_PROVIDER_IDS;
+  for (const agentConfig of agentConfigurations) {
+    if (!whiteListedProviders.includes(agentConfig.model.providerId)) {
+      yield {
+        type: "agent_disabled_error",
+        created: Date.now(),
+        configurationId: agentConfig.sId,
+        error: {
+          code: "provider_disabled",
+          message: `The model provider for agent configuration ${agentConfig.sId} is currently disabled.`,
```
Comment (spolu) [inline front/lib/api/assistant/conversation.ts:661]: Let's use the assistant name and say that it was disabled by the workspace admin:

Assistant {assistant name} is based on a model that was disabled by your workspace admin. Please edit the assistant to use another model (advanced settings in the Instructions panel).

**Review by spolu — APPROVED** [review]
LGTM modulo nit on the message

File: front/lib/api/assistant/conversation.ts (hunk 1 above)
Comment (JulesBelveze) [reply front/lib/api/assistant/conversation.ts]: 👍🏼

Comment (JulesBelveze) [reply front/lib/api/assistant/conversation.ts]: @spolu also added the check in `editUserMessage`

**Review by spolu — APPROVED** [review]

**Pull request merged**